src/actions/guides/frontend/internationalization.cr

```diff
@@ -9,133 +9,151 @@ class Guides::Frontend::Internationalization < GuideAction
+    This will geneate:
 
-    > **NOTE:** If you missed a key and get the above error **YOU MUST RESTART LUCKY** to reload its config.
+    **1. An initializer at `config/rosetta.cr` with the following content:**
 
-    ```yaml
-    # config/locales/en.yml
-    en:
-      action:
-        save_success: "The record has been saved"
-        update_success: "The record has been updated"
-        delete_success: "Deleted the record"
-        index_title: "All Records"
-        create_new: "New Record"
-        new: New
-        edit: Edit
-        delete: Delete
-        confirm: "Are you sure?"
-        update: Update
-        updating: "Updating..."
-        save: Save
-        saving: "Saving..."
-        back: Back
-        back_to_index: "Back to All"
-      auth:
-        sign_in: "Sign In"
-        sign_in_success: "You're now signed in"
-        sign_in_failure: "Sign in failed"
-        sign_up: "Sign up"
-        sign_up_success: "Thanks for signing up"
-        sign_up_failure: "Couldn't sign you up"
-        sign_out: "Sign Out"
-        signed_out: "You have been signed out"
-        pwd_update: "Update Password"
-        pwd_update_success: "Your password has been reset"
-        pwd_reset: "Password Reset"
-        pwd_reset_request: "Reset your Password"
-        pwd_reset_req_success: "You should receive an email on how to reset your password shortly"
-      auth_token:
-        not_authenticated: "Not Authenticated."
-        invalid: "The provided authentication token was incorrect."
-        missing: "An authentication token is required. Please include a token in an 'auth_token' param or 'Authorization' header."
-      default:
-        page_name: Welcome
-      error:
-        title: "Something went wrong"
-        try_home: "Try heading back to home"
-        locked_out: "Locked-out"
-        auth_incorrect: "is wrong"
-        form_not_valid: "It looks like the form is not valid"
-        not_in_system: "is not in our system"
-      user:
-        email: E-Mail
-        profile: Profile
-        next: "Next, you may want to"
-        auth_guides: "Check out the 'Authentication Guides'"
-        modify_page: "Modify this page"
-        after_signin: "Change where you go after sign in"
-        preferred_language: "Preferred Language"
+    ```cr
+    Rosetta::DEFAULT_LOCALE = :en
+    Rosetta::AVAILABLE_LOCALES = %i[en]
+    Rosetta::Backend.load("config/rosetta")
     ```
 
-    ## Step 3 - Configure i18n within Lucky
-
-    The i18n needs to know the location of your language files.
-    > NOTE: This config file is only executed at load, so all language changes require a server restart.
+    Rosetta includes an integration macro for Lucky to include the
+    `Rosetta::Translatable` module everywhere translations are needed. Add the
+    following line to the initializer and you're good to go:
```

Comment:
is this saying this command also need to go in `config/rosetta.cr`? If so, does it matter where?

Reply:
Good point, it's a bit unclear so I'll fix that.